src/Instance/HoldingMovementList/HoldingMovement/HoldingMovement.js

```diff
@@ -0,0 +1,139 @@
+import React, {
+  useMemo,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  IfPermission,
+} from '@folio/stripes/core';
+import {
+  Accordion,
+  Row,
+  Col,
+  Button,
+  Checkbox,
+} from '@folio/stripes/components';
+
+import {
+  callNumberLabel
+} from '../../../utils';
+import { ItemsListContainer } from '../../ItemsList';
+
+const HoldingMovement = ({
+  holding,
+  referenceData,
+  onViewHolding,
+  onAddItem,
+
+  draggable,
+  droppable,
+  ifItemsDragSelected,
+  selectItemsForDrag,
+  selectHoldingsForDrag,
+  ifHoldingDragSelected,
+  getDraggingItems,
+}) => {
+  const { locationsById } = referenceData;
+  const labelLocation = holding.permanentLocationId ? locationsById[holding.permanentLocationId].name : '';
+
+  const viewHoldings = useCallback(() => {
```

Comment:
it looks like in current way there is no point in useCallback, since you can use `onViewHolding` directly, doesn't it?